cost/aws/elb/clb_unused/AWS_Delete_Unused_CLB.pt

```diff
@@ -0,0 +1,243 @@
+name "AWS Delete Unused Elastic Load Balancers (CLB)"
+rs_pt_ver 20180301
+type "policy"
+short_description "Report and remediate any  ELB (CLB) that are not currently in use. \n See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/aws/elb/clb_unused) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+category "Cost"
+severity "low"
+
+##################
+# User inputs    #
+##################
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses of the recipients you wish to notify"
+end
+
+parameter "param_exclude_tags" do
+  type "list"
+  label "List of one or more Tags that will exclude ELB(CLB) from actions being taken. Format: Key=Value"
+  allowed_pattern /([\w]?)+\=([\w]?)+/
+end
+
+auth "aws", type: "aws" do
+  version "4"
+  service "elasticloadbalancing"
+  access_key cred("AWS_ACCESS_KEY_ID")
+  secret_key cred("AWS_SECRET_ACCESS_KEY")
+end
+
+#https://docs.aws.amazon.com/elasticloadbalancing/2012-06-01/APIReference/API_DescribeLoadBalancers.html
+datasource "ds_aws_clb_list" do
```

Comment:
Line indentation is not uniform in this datasource.

Reply:
Line indentation is taken care in this datasource now.